@navikt/ds-css/accordion/index.css

```diff
@@ -0,0 +1,75 @@
+.ReactCollapse--collapse {
+  transition: height 300ms ease-out;
+}
+.navds-accordion {
+  --accordion-border-radius: 4px;
+  position: relative;
+  background: var(--navds-color-background);
+  border-radius: var(--accordion-border-radius);
+  border: var(--navds-border-default);
+}
+
+.navds-accordion:hover {
+  box-shadow: var(--navds-shadow-hover);
+  border: var(--navds-panel-hover);
+}
+
+.navds-accordion:focus,
+.navds-accordion__button:focus {
+  box-shadow: var(--navds-shadow-focus);
+  outline: none;
+}
+
+.navds-accordion__chevron {
+  --chevron-rotation: 0deg;
+  width: 1.25em;
+  height: 1.25em;
+  transition: 300ms transform;
+  transform: rotateZ(var(--chevron-rotation));
+}
+.navds-accordion__chevron--up {
+  --chevron-rotation: -180deg;
+}
+.navds-accordion__chevron--down {
+  --chevron-rotation: 0deg;
+}
+
+.navds-accordion__button {
+  background-color: var(--navds-color-background);
+  display: block;
+  padding: 1rem;
+  border-radius: var(--accordion-border-radius);
+  background: transparent;
+  border: 0;
+  text-align: left;
+  width: 100%;
+  margin-bottom: 0;
+  cursor: pointer;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.navds-accordion__button .navds-accordion__title {
+  font-size: var(--navds-font-size-large);
+  line-height: var(--navds-font-line-height-xs);
+  font-weight: var(--navds-font-weight-bold);
+  color: var(--navds-color-text-primary);
```

Comment:
Trenger vi sette denne? Burde vi i så fall sette den for andre komponenter også?

Reply:
Trenger nok ikke å være med nei